index.html

```diff
@@ -0,0 +1,120 @@
+                    <input type="number" step=1 value=0 name="Jpos_X_Ch" id="Jpos_X_Ch" placeholder="Jpos X Ch" />
+                </label>
+                <br>
+                <label>Jpos Y Ch :
+                    <input type="number" step=1 value=0 name="Jpos_Y_Ch" id="Jpos_Y_Ch" placeholder="Jpos Y Ch" />
+                </label>
+                <br>
+                <label>Mpos X Ch :
+                    <input type="number" step=1 value=0 name="Mpos_X_Ch" id="Mpos_X_Ch" placeholder="Mpos X Ch" />
+                </label>
+                <br>
+                <label>Mpos Y Ch :
+                    <input type="number" step=1 value=0 name="Mpos_Y_Ch" id="Mpos_Y_Ch" placeholder="Mpos Y Ch" />
+                </label>
+                <br>
+                <label>Lick Ch :
+                    <input type="number" step=1 value=0 name="Lick_Ch" id="Lick_Ch" placeholder="Lick Ch" />
+                </label>
+                <br>
+                <label>Rew Ch :
+                    <input type="number" step=1 value=0 name="Rew_Ch" id="Rew_Ch" placeholder="Rew Ch" />
+                </label>
+                <br>
+                <label>Trial Start Ch :
+                    <input type="number" step=1 value=0 name="Trial_Start_Ch" id="Trial_Start_Ch" placeholder="Trial Start Ch" />
+                </label>
+                <br>
+                <label>Trial Event Ch :
+                    <input type="number" step=1 value=0 name="Trial_Event_Ch" id="Trial_Event_Ch" placeholder="Trial Event Ch" />
+                </label>
+                <br>
+                <label>Stim1 :
+                    <input type="text" name="Stim1" id="Stim1" placeholder="Stim1" />
+                </label>
+                <br>
+                <label>Stim2 :
+                    <input type="text" name="Stim2" id="Stim2" placeholder="Stim2" />
+                </label>
+                <br>
+                <label>Trial Start :
+                    <input type="number" step="any" value=0 name="Trial_Start" id="Trial_Start" placeholder="Trial Start" />
+                </label>
+                <br>
+                <label>Operant Event1 :
+                    <input type="number" step="any" value=0 name="Operant_Event1" id="Operant_Event1" placeholder="Operant_Event1" />
+                </label>
+                <br>
+                <label>Operant Event2 :
+                    <input type="number" step="any" value=0 name="Operant_Event2" id="Operant_Event2" placeholder="Operant_Event2" />
+                </label>
+                <br>
+                <label>Water Port :
+                    <input type="number" step="any" value=0 name="Water_Port" id="Water_Port" placeholder="Water_Port" />
+                </label>
+                <br>
+                <label>Laser Power :
+                    <input type="number" step="any" value=0 name="LaserPower" id="LaserPower" placeholder="LaserPower" />
+                </label>
```

Comment:
Could be handy to have a notes section for miscellaneous info about how the recording is done.